src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -593,7 +617,7 @@ void GcsPlacementGroupManager::OnNodeDead(const NodeID &node_id) {
       // creating until a node with the resources is added. we will solve it in next pr.
       if (iter->second->GetState() != rpc::PlacementGroupTableData::RESCHEDULING) {
         iter->second->UpdateState(rpc::PlacementGroupTableData::RESCHEDULING);
-        pending_placement_groups_.emplace_front(iter->second);
+        AddToPendingQueue(iter->second, 0);
```

Comment:
Can you add a comment to the argument 0? 
```suggestion
        AddToPendingQueue(iter->second, /*[arg_name]*/0);
```